.github/workflows/cd.yml

```diff
@@ -1,10 +1,18 @@
 name: cd
 
-on: [push, pull_request, workflow_dispatch]
+on:
+    push:
+        branches:
+            - main
+    pull_request:
+        branches:
+            - main
+    workflow_dispatch:
 
 jobs:
     cd:
         runs-on: ${{ matrix.os }}
+        if: github.ref == 'refs/heads/master'
```

Comment:
We probably don't need this, if we have the rules above.

Reply:
My bad, I forgot to remove this.